paasta_tools/check_chronos_jobs.py

```diff
@@ -152,31 +144,65 @@ def message_for_status(status, service, instance, cluster):
         raise ValueError('unknown sensu status: %s' % status)
 
 
-def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster, job_state_pairs):
-    if len(job_state_pairs) > 1:
-        sensu_status = pysensu_yelp.Status.UNKNOWN
-        output = (
-            "Unknown: somehow there was more than one enabled job for %s%s%s.\n"
-            "Talk to the PaaSTA team as this indicates a bug." % (service, utils.SPACER, instance)
-        )
-    elif len(job_state_pairs) == 0:
+class TZ(tzinfo):
+
+    def utcoffset(self, dt):
+        return timedelta(minutes=0)
+
+    def dst(self, dt):
+        return timedelta(minutes=0)
+
+
+utc = TZ()
+
+
+def job_is_stuck(last_run_iso_time, interval_in_seconds):
+    if last_run_iso_time is None or interval_in_seconds is None:
+        return False
+    last_run_datatime = isodate.parse_datetime(last_run_iso_time)
+    return last_run_datatime + timedelta(seconds=interval_in_seconds) < datetime.now(utc)
+
+
+def message_for_stuck_job(service, instance, cluster, last_run_iso_time, interval_in_seconds):
+    return ("Job %(service)s%(separator)s%(instance)s hasn't run since %(last_run)s,"
+            " and is configured to run every %(interval).1f minutes.\n\n"
```

Comment:
There isn't really any action here for the user - maybe suggest they speak to someone on the PaaSTA team?